stubs/pyOpenSSL/OpenSSL/crypto.pyi

```diff
@@ -129,7 +129,7 @@ class X509Store:
     def __init__(self) -> None: ...
     def add_cert(self, cert: X509) -> None: ...
     def add_crl(self, crl: CRL) -> None: ...
-    def load_locations(self, cafile: Text | bytes, capath: Text | bytes | None = ...) -> None: ...
+    def load_locations(self, cafile: str | bytes, capath: str | bytes | None = ...) -> None: ...
```

Comment:
`cafile` and `capath` should be annotated with `_typeshed.StrOrBytesPath`. They get passed to `_path_bytes` which calls `os.fspath`.

Reply:
I added the suggestions